functions/Invoke-DbaDbShrink.ps1

```diff
@@ -222,16 +222,16 @@ function Invoke-DbaDbShrink {
                     $desiredFileSize = $spaceUsed + $desiredSpaceAvailable
 
                     Write-Message -Level Verbose -Message "File: $($file.Name)"
-                    Write-Message -Level Verbose -Message "Initial Size (KB): $([int]$startingSize)"
-                    Write-Message -Level Verbose -Message "Space Used (KB): $([int]$spaceUsed)"
-                    Write-Message -Level Verbose -Message "Initial Freespace (KB): $([int]$spaceAvailable)"
-                    Write-Message -Level Verbose -Message "Target Freespace (KB): $([int]$desiredSpaceAvailable)"
-                    Write-Message -Level Verbose -Message "Target FileSize (KB): $([int]$desiredFileSize)"
+                    Write-Message -Level Verbose -Message "Initial Size (KB): $([dbasize]$startingSize)"
```

Comment:
Bit more efficient if we set this when the variables referenced are declared, line 218-222

Reply:
I agree, great catch